package.json

```diff
@@ -131,11 +131,9 @@
 		"mkdirp": "0.5.1",
 		"node-sass": "4.12.0",
 		"node-watch": "0.6.0",
-		"npm-package-json-lint": "4.0.2",
```

Comment:
Those deps shouldn't be included here, I forgot to remove them. They are hoisted from `@wordpress/scripts`.